src/Tribe/REST/V1/Post_Repository.php

```diff
@@ -0,0 +1,319 @@
+				'ID'                     => $organizer->ID,
+				'author'                 => $organizer->post_author,
+				'date'                   => $organizer->post_date,
+				'date_utc'               => $organizer->post_date_gmt,
+				'modified'               => $organizer->post_modified,
+				'modified_utc'           => $organizer->post_modified_gmt,
+				'link'                   => get_the_permalink( $organizer->ID ),
+				'title'                  => trim( apply_filters( 'the_title', $organizer->post_title ) ),
+				'description'            => trim( apply_filters( 'the_content', $organizer->post_content ) ),
+				'excerpt'                => trim( apply_filters( 'the_excerpt', $organizer->post_excerpt ) ),
+				'featured_image'         => get_the_post_thumbnail_url( $organizer->ID, 'full' ),
+				'featured_image_details' => $this->get_featured_image_details( $organizer->ID ),
+				'phone'                  => isset( $meta['_OrganizerPhone'] ) ? $meta['_OrganizerPhone'] : '',
+				'website'                => isset( $meta['_OrganizerWebsite'] ) ? $meta['_OrganizerWebsite'] : '',
+				'email'                  => isset( $meta['_OrganizerEmail'] ) ? $meta['_OrganizerEmail'] : '',
+			);
+
+
+			/**
+			 * Filters the data that will be returnedf for a single organizer.
+			 *
+			 * @param array   $data  The data that will be returned in the response.
+			 * @param WP_Post $event The requested organizer.
+			 */
+			$this_data = apply_filters( 'tribe_rest_organizer_data', array_filter( $this_data ), $organizer );
+
+			$data[] = $this_data;
+		}
+
+		/**
+		 * Filters the data that will be returnedf for all the organizers of an event.
+		 *
+		 * @param array   $data            The data that will be returned in the response; this is
+		 *                                 an array of organizer data arrays.
+		 * @param WP_Post $event           The requested event.
+		 */
+		$data = apply_filters( 'tribe_rest_event_organizer_data', array_filter( $data ), get_post( $event_or_organizer_id ) );
+
+		$data = array_filter( $data );
+
+		return $single ? $data[0] : $data;
+	}
+
+	protected function get_categories( $event_id ) {
+		return array();
+	}
+
+	protected function get_tags( $event_id ) {
+		return array();
+	}
+
+	/**
+	 * @param int $id The event post ID.
+	 */
+	protected function get_featured_image_details( $id ) {
+		$thumbnail_id = get_post_thumbnail_id( $id );
+
+		if ( empty( $thumbnail_id ) ) {
+			return array();
+		}
+
+		$metadata = wp_get_attachment_metadata( $thumbnail_id );
+		$data = array( 'ID' => $thumbnail_id );
+
+		if ( false !== $metadata
+		     && isset( $metadata['image_meta'] )
+		     && isset( $metadata['file'] )
+		     && isset( $metadata['sizes'] )
+		) {
+			unset( $metadata['image_meta'], $metadata['file'] );
+			$metadata['url'] = wp_get_attachment_image_src( $thumbnail_id, 'full' )[0];
+
+			foreach ( $metadata['sizes'] as $size => &$meta ) {
+				$meta['url'] = wp_get_attachment_image_src( $thumbnail_id, $size )[0];
```

Comment:
Same thought here regarding array access on the function not being supported until PHP 5.4+